core/src/main/java/org/elasticsearch/index/search/MultiMatchQuery.java

```diff
@@ -111,11 +111,11 @@ public QueryBuilder(boolean groupDismax, float tieBreaker) {
             this.tieBreaker = tieBreaker;
         }
 
-        public List<Query> buildGroupedQueries(MultiMatchQueryBuilder.Type type, Map<String, Float> fieldNames, Object value, String minimumShouldMatch) throws IOException{
+        public List<Query> buildGroupedQueries(MultiMatchQueryBuilder.Type type, Map<String, Float> fieldNamesWithBoost, Object value, String minimumShouldMatch) throws IOException{
```

Comment:
I looked up the usage of this method and found out the Float is actually the boost for a field, which was not obvious before I looked it up. So I renamed it to this if that's okay.

The code that calls this method also uses `fieldNames` without indicating the Float is the boost of the field.